azure/table_azure_storage_share_file.go

```diff
@@ -0,0 +1,277 @@
+package azure
+
+import (
+	"context"
+	"strings"
+
+	"github.com/Azure/azure-sdk-for-go/services/storage/mgmt/2019-06-01/storage"
```

Comment:
The current version for storage is 2021-04-01, can we try with it?

Reply:
With version `github.com/Azure/azure-sdk-for-go/services/storage/mgmt/2021-04-01/storage`, the get call has been removed.